target/linux/ar71xx/config-4.9

```diff
@@ -198,6 +198,7 @@ CONFIG_ATH79=y
 # CONFIG_ATH79_MACH_TL_WA701ND_V2 is not set
 # CONFIG_ATH79_MACH_TL_WA7210N_V2 is not set
 # CONFIG_ATH79_MACH_TL_WA801ND_V3 is not set
+CONFIG_ATH79_MACH_TL_WA801ND_V4=y
```

Comment:
Use target/linux/ar71xx/tiny/config-default for 4M device.

Reply:
You are right. Will change this.